swap_meet/clothing.py

```diff
@@ -0,0 +1,27 @@
+from swap_meet.item import Item
+
+# ---- Wave 5 ----- #
+class Clothing(Item):
+    '''
+    Represents a clothing Item
+    Attributes: 
+        category (str)
+        condition (int/float)
+        age (int/float)
+    '''
+    def __init__(self, category="", condition=0, age=0): 
+        '''
+        Inherits methods of the parent class Item
+        Parameters: 
+            category ("Clothing")
+            condition (int/float)
+            age (int/float)
+        '''
+        super().__init__("Clothing", condition, age)
+
+    def __str__(self): 
+        '''
+        Stringifies an instance of Clothing 
+        '''
+        return "The finest clothing you could wear."
```

Comment:
👍  this looks great! let's try and clean up some of these docstring because there are a lot of them and it's harder to read it.


```suggestion
'''
Represents a clothing Item and Inherits methods of the parent class Item
'''
class Clothing(Item):
    def __init__(self, condition=0, age=0): 
        super().__init__("Clothing", condition, age)

    def __str__(self): 
        return "The finest clothing you could wear."
```

so, because Clothing class category will always be "clothing," we don't need to give the user the ability to change that in this situation.

also, I've cleaned up the docstring and fir the important bits under one docstring